pkg/controller/kms/key/setup.go

```diff
@@ -0,0 +1,330 @@
+package key
+
+import (
+	"context"
+	"sort"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/kms"
+	svcsdkapi "github.com/aws/aws-sdk-go/service/kms/kmsiface"
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/pkg/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	svcapitypes "github.com/crossplane/provider-aws/apis/kms/v1alpha1"
+	aws "github.com/crossplane/provider-aws/pkg/clients"
```

Comment:
```suggestion
	"context"
	"sort"

	svcsdk "github.com/aws/aws-sdk-go/service/kms"
	svcsdkapi "github.com/aws/aws-sdk-go/service/kms/kmsiface"
	"github.com/pkg/errors"
	ctrl "sigs.k8s.io/controller-runtime"
	
	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
	"github.com/crossplane/crossplane-runtime/pkg/event"
	"github.com/crossplane/crossplane-runtime/pkg/logging"
	"github.com/crossplane/crossplane-runtime/pkg/meta"
	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
	"github.com/crossplane/crossplane-runtime/pkg/resource"

	svcapitypes "github.com/crossplane/provider-aws/apis/kms/v1alpha1"
	aws "github.com/crossplane/provider-aws/pkg/clients"
```

We usually have a grouping like `std -> non-crossplane-org -> crossplane-org -> this-repo`